src/algebra/commute.lean

```diff
@@ -260,9 +261,22 @@ instance centralizer.is_submonoid : is_submonoid (centralizer a) :=
 { one_mem := commute.one_right a,
   mul_mem := λ _ _, commute.mul_right }
 
+/-- Centralizer of an element of a monoid `a` is the submonoid of elements that commute with `a`. -/
+@[to_additive hmm] def centralizer.submonoid : submonoid M :=
```

Comment:
Is this a typo `@[to_additive hmm] `? I think maybe `submonoid.centralizer` is a better name, because then you can open the namespace and have a short name.